docs/deployment_patterns/how_to_use_great_expectations_in_deepnote.md

```diff
@@ -7,15 +7,15 @@ import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
 _This piece of documentation was authored by [Allan Campopiano](https://www.linkedin.com/in/allan-campopiano-703394120)._
 
-This guide will help you get started with Great Expectations on Deepnote. You will learn how to validate columns in a Pandas DataFrame, 
+This guide will help you get started with Great Expectations on Deepnote. You will learn how to validate columns in a Pandas DataFrame,
```

Comment:
My editor strips unnecessary whitespace so please be aware that some of the diffs are just this